lib/cru_lib/global_registry_methods.rb

```diff
@@ -43,7 +43,7 @@ def attributes_to_push(*args)
         @attributes_to_push = {}
         attributes_to_push['client_integration_id'] = id unless self.class.skip_fields_for_gr.include?('client_integration_id')
         attributes_to_push['client_updated_at'] = updated_at if respond_to?(:updated_at)
-        attributes.collect {|k, v| @attributes_to_push[k.underscore] = self.class.gr_value(k.underscore, v)}
+        attributes.collect {|k, v| @attributes_to_push[k.underscore] = self.class.gr_value(k.underscore, v)} # MK: shouldn't that be .each() ?
```

Comment:
This is unrelated, just a question I had pending I thought I may as well ask while I have the chance.